src/yunohost/tests/test_apps.py

```diff
@@ -60,6 +61,33 @@ def clean():
     os.system("systemctl reset-failed nginx")  # Reset failed quota for service to avoid running into start-limit rate ?
     os.system("systemctl start nginx")
 
+    # Clean permission
```

Comment:
I can't suggest in the right place but I think we should make sure that legacy_app__2 is removed too (multi instance tests):

line 41:
```
    if _is_installed("legacy_app__2"):
        app_remove("legacy_app__2")
```